tests/network-tests/src/Fixture.ts

```diff
@@ -138,7 +138,7 @@ export abstract class StandardizedFixture extends BaseQueryNodeFixture {
     })
   }
 
-  private flattenExtrinsics(
+  protected flattenExtrinsics(
```

Comment:
I think this method is actually pretty redundant, as one can just do `extrinsics.flat()` (which I probably didn't know back then)